_posts/current/blog/2021-02-23-upstream-camera-support-for-qualcomm-platforms.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: Upstream camera support for Qualcomm platforms
+description: "Since 2017 Linaro has worked with Qualcomm to enable camera
+  support on their platforms. In this blog, Linaro engineer Robert Foss talks
+  about how the Open Source CAMSS driver (with the same name) now supports the
+  next generation of ISP (image signal processor) artchitecture and what
+  improvements this entails. "
+date: 2021-02-23 01:49:15
+image: /assets/images/content/code-background_1.jpg
+tags:
```

Comment:
Could you please update these tags based on available tags in https://www.linaro.org/assets/json/tags.json ?

Reply:
That's weird, I added tags. Ok let me take a look.